### ISSUE TITLE
No scale layers after batchnorm

### ISSUE THREAD

**Issue by mprat** [cvjena/cnn-models#7]
Hello! Thanks for these pre-trained models. I was wondering why you did not include the Scale layer after the BN layer in your AlexNet model. None of the convolutional layers have a bias term, so because there is no scale layer added, it turns out there are no bias terms throughout the network. Was that intentional?

**Comment by MarcelSimon** [comment]
Hi mprat! The conv layers still have bias term, there is no `bias_term: false` as far as I can see. Why do you think there is no bias term?
It might be worth a try to add the scale layers though. Leaving out the scale layer was merely a design choice at the time of creation. However, it might give another percent accuracy. Feel free to try :)

**Comment by MarcelSimon** [comment]
I quickly ran a training with additional scale layers added after each batch norm. The final validation accuracy is almost identical to the current net.
